<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimeEpisodeNewDialog</class>
 <widget class="QDialog" name="AnimeEpisodeNewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>197</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>新增话</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0">
   <item>
    <layout class="QHBoxLayout" name="hLayout_EpisodePreview" stretch="0,1">
     <item>
      <widget class="QLabel" name="label_EpisodePreviewText">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>序号预览</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_EpisodePreview">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string notr="true">01</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout_Episode">
     <item>
      <widget class="QRadioButton" name="radioButton_OneEpisode">
       <property name="text">
        <string>单话</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_Episode</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Episode">
       <property name="text">
        <string>序号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Episode">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string notr="true">01</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacer_Episode">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout_MoreEpisode">
     <item>
      <widget class="QRadioButton" name="radioButton_MoreEpisode">
       <property name="text">
        <string>多话</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup_Episode</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_EpisodeCount">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>话数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_EpisodeCount">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string notr="true">12</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_EpisodeFormat">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>格式</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_EpisodeFormat">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>[%ep%]代表序号数字部分</string>
       </property>
       <property name="text">
        <string notr="true">[%ep%]</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_EpisodeBegin">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>开始序号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_EpisodeBegin">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string notr="true">1</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_EpisodeFillZeroNum">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>序号数字位数</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_EpisodeFillZeroNum">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>不补零</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2位补零</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3位补零</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4位补零</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="hSpacer_MoreEpisode">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout_Title">
     <item>
      <widget class="QLabel" name="label_Title">
       <property name="text">
        <string>标题</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Title">
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout_ABC">
     <item>
      <spacer name="hSpacer_ABC">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Tag1">
       <property name="text">
        <string>Tag1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Tag2">
       <property name="text">
        <string>Tag2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_Tag3">
       <property name="text">
        <string>Tag3</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout_OkCancel">
     <item>
      <spacer name="hSpacer_OkCancel">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Ok">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_Cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_Episode"/>
 </buttongroups>
</ui>
